src-tauri/src/manager.rs

```diff
@@ -145,6 +172,10 @@ pub fn start_manager() -> (Sender<WatcherMessage>, Arc<Mutex<ManagerState>>) {
     for watcher in autostart_watchers.iter() {
         start_watcher(watcher, tx.clone());
     }
+    let tx_clone = tx.clone();
+    thread::spawn(move || {
+        wrapper_start_watcher(tx_clone);
+    });
```

Comment:
This swaps a thread that lives for the entirety of the program.

Reply:
It felt silly to start a thread whose sole purpose it was to start other threads. I refactored this in [`bfaa31b`](https://github.com/ActivityWatch/aw-tauri/pull/9/commits/bfaa31bbf40d7b2bd302ff7c87a8a1d4036f85f4), also removing the need for the CONDVAR's